js/runner.go

```diff
@@ -765,6 +766,16 @@ func (u *ActiveVU) RunOnce() error {
 	ctx, cancel := context.WithCancel(u.RunContext)
 	defer cancel()
 	u.moduleVUImpl.ctx = ctx
+
+	eventIterData := event.IterData{
+		Iteration:    uint64(u.iteration),
```

Comment:
Out of curiosity, why use `uint64` for `Iteration` field in `IterData` if the same field in `State` is represented as `int64`? For what I know, `Iteration` is an unsigned monotonic int, so I guess it makes sense to use `uint64`, but would it be better to be coherent with the previous implementation?

No big deal in any case, but when integrating with these events I find myself having to convert the `State` iteration from `int64` to `uint64` numerous times :sweat_smile:

Reply:
Yeah, it's better to be consistent than correct in this case, so I'll change it.

Reply:
Fixed in the last push.